client/signup/controller.js

```diff
@@ -233,31 +233,12 @@ export default {
 
 		// See: 1113-gh-Automattic/experimentation-platform for details.
 		if ( isOnboardingFlow || isOnboardingGuidedFlow( flowName ) ) {
```

Comment:
These checks now are basically the same

Reply:
Let's keep them for now, as we may need to revert the change of guided to be 'onboarding'. 

https://github.com/Automattic/dotcom-forge/issues/7840